src/mm2_game.h

```diff
@@ -1471,6 +1496,32 @@ namespace MM2
     };
     ASSERT_SIZEOF(mmWaypoints, 0x90);
 
+    enum HudmapIconColor {
+        Black,
+        BrightRed,
+        Blue,
+        Green,
+        Red,
+        Yellow,
+        Orange,
+        Purple,
+        Cyan,
+        Pink,
+        White,
+        LightOrange,
+        LightGreen,
+        DarkBlue,
+        DarkRed,
+        Grey,
+    };
+
+    enum HudmapMode {
+        Off,
+        Small,
+        HalfScreen,
+        FullScreen,
+    };
```

Comment:
Some of these enum names are very generic and might clash with other things at some point.
It would help to add a prefix to the names to differentiate them (i.e `HudmapMode_Off` or something).

Reply:
Well, I'm gonna move them inside the class itself instead.